lib/elixir/lib/kernel.ex

```diff
@@ -5462,6 +5462,69 @@ defmodule Kernel do
     raise ArgumentError, "modifier must be one of: s, a, c"
   end
 
+  @doc """
+  Handles the `~PID` sigil.
+
+  This function allows creating arbitrary PIDs, even the ones that don't actually exist in the VM.
+  As such, it should only be used for debugging purposes.
+  """
+  @doc since: "1.11.0"
+  def sigil_PID(string, modifiers)
+
+  def sigil_PID(string, []) do
```

Comment:
If you're using the bodiless clause for having a nice argument name for the `modifiers` argument, you can do:

```suggestion
  def sigil_PID(string, [] = _modifiers) do
```

Shaves off a couple of lines and achieves the exact same goal. Applies to all the new functions.